pkg/audit/disqualification_test.go

```diff
@@ -118,42 +118,42 @@ func TestDisqualifiedNodesGetNoDownload(t *testing.T) {
 	testplanet.Run(t, testplanet.Config{
 		SatelliteCount: 1, StorageNodeCount: 4, UplinkCount: 1,
 	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
-		satellite := planet.Satellites[0]
-		upl := planet.Uplinks[0]
+		satellitePeer := planet.Satellites[0]
+		uplinkPeer := planet.Uplinks[0]
```

Comment:
`upl` yeah doesn't pass the telephone test, so there should be a better variable. `uplinkPeer` seems long similiarly to `satellitePeer`, but I don't have a better suggestion for it yet.

Reply:
No idea of a better name, but I have this in mind `ulnk` however I don't call it for being better than the ones used.